### ISSUE TITLE
Double[] is not correctly colored as a method property

### ISSUE THREAD

**Issue by duemoselund** [OmniSharp/omnisharp-vscode#1812]
## Environment data
`dotnet --info` output:
VS Code version:
C# Extension version:

## Steps to reproduce
Write a method that includes double array as a property

## Expected  behavior
That the colors were consistent for all data types

## Actual behavior
Double[] as a property is not correctly colored.
![parameter color error](https://user-images.githubusercontent.com/11139391/32103009-00180574-bb1f-11e7-9a0b-ea8196959576.PNG)

**Comment by rchande** [comment]
@DustinCampbell this sounds like it's probably an issue with the classifier regexes, no? I can take this if you remind me what repo they're in.

**Comment by DustinCampbell** [comment]
This issue was moved to dotnet/csharp-tmLanguage#86

**Comment by DustinCampbell** [comment]
Thanks for reporting this @duemoselund! I've got a feeling that it's related to the multi-dimensional array in the first parameter. My guess is that the `,` in the multi-dimensional array type is being treated as the comma for the next parameter in the list. I've moved this to the repo for the C# TextMate grammar.

@rchande: FYI that I used https://github-issue-mover.appspot.com/ to move the issue.
